### PR TITLE
Merging master

### PR DESCRIPTION
**Pull request by chriskavanagh97**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
...
+
+* B = 4, 
+* W = 12, 
+* M = 6 
+
+So, the task duration is calculated as:
+
+(4 +12 + 14)/6 = 7. **(** *All Numbers should be rounded to the nearest integer* **)** 
+
+Format: ![Alt Text](Doccuments/threepoint.png)
```
Comment (mikeygillen) [inline README.md:26]: Picture not displaying

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+
+__Let’s show an example:__
+
+* B = 4, 
+* W = 12, 
+* M = 6 
+
+So, the task duration is calculated as:
+
+(4 +12 + 14)/6 = 7. **(** *All Numbers should be rounded to the nearest integer* **)**
```
Comment (mikeygillen) [inline README.md:24]: No need for example? Or at least shorten it down a bit.

**Comment by harrythomas1998** [comment]
Would read better if you gave a description of the formula, so people know exactly what it is

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+
+**Task Duration** == *(B + W +(4 * M)) / 6*
+
+__Let’s show an example:__
+
+* B = 4, 
+* W = 12, 
+* M = 6 
+
+So, the task duration is calculated as:
```
Comment (mikeygillen) [inline README.md:22]: No need for example?

**Review by alinakazzaa — COMMENTED** [review]
Needs links to resources for further reading

File: README.md (hunk 2 above)
Comment (mikeygillen) [inline README.md:24]: Remove sum but keep tip about rounding to int

**Pull request closed without merging**